src/scss/blocks/_banner.scss

```diff
@@ -9,45 +7,36 @@
   flex-wrap: wrap;
   align-items: flex-start;
   font-size: get-size('size-0');
-}
-
-.banner__content {
-  @include apply-utility('leading', 'short');
-}
-
-.banner__actions {
-  $banner-actions-button-padding-x: 1rem;
-  $banner-actions-button-padding-y: 0.8rem;
-
-  flex: none; /// We don't want this to shrink
-  margin-inline-start: auto;
-  margin-block-start: auto;
 
-  button.button {
-    padding: $banner-actions-button-padding-y $banner-actions-button-padding-x;
-
-    @include apply-utility('text', 'size-0');
+  &__content {
+    @include apply-utility('leading', 'short');
   }
 
-  /// Let whatever button element's current hover state happen
-  button:not(:hover) {
-    background: transparent;
+  &__img {
+    max-width: 2.5em;
+    max-height: 2.5em;
+    /// A slightly lighter background, regardless of current color
+    background: rgba(255, 255, 255, 0.5);
   }
 }
 
-/// If the content has a cluster on it,
-/// we need to make some adjustments
-.banner__content.cluster {
-  --gutter: #{$global-gutter};
+a:visited {
```

Comment:
Wouldn't this be a site-wide change?

Reply:
argh that was bad. Added a class to the component and fixed the SCSS.